app/components/SideNav.tsx

```diff
@@ -130,7 +131,9 @@ export default class SideNav extends Vue {
     if (
       getOS() === OS.Windows &&
       this.userService.isLoggedIn &&
-      this.incrementalRolloutService.views.featureIsEnabled(EAvailableFeatures.highlighter)
+      this.incrementalRolloutService.views.featureIsEnabled(EAvailableFeatures.highlighter) &&
+      // TODO: Remove via bundle when v1 is complete and ready to start rolling out
+      (Utils.isPreview() || Utils.isDevMode())
```

Comment:
the feature is always enabled in DevMode

Reply:
Yeah the purpose of this check is to ensure it's always disabled in production (for now).

Reply:
We don't have a `Utils.isProductionButNotPreview()` check so that's what this is effectively.